code/modules/clothing/spacesuits/miscellaneous.dm

```diff
@@ -42,6 +43,54 @@ Contains:
 	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/deathsquad
 	dog_fashion = /datum/dog_fashion/back/deathsquad
 
+/obj/item/clothing/head/helmet/space/hardsuit/flamer
+	name = "MK.II FLAME Helmet"
+	desc = "A space proof, fixium-reinforced SWAT mask. The extra reinforcements means it has to be a helmet, though."

```

Comment:
by the way, this lore is correct. Fixium is in the description of the bomb test site indestructable wall